#!/usr/bin/env python
PACKAGE = "mrm_env"
from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("joint1", double_t, 0, "q1", 0,  -3.14, 3.14)
gen.add("joint2", double_t, 0, "q2",    0, -1.57,   1.57)
gen.add("joint3", double_t, 0, "q3", 0,  -1.57, 1.57)
gen.add("joint4", double_t, 0, "q4",    0, 0,   1.57)
gen.add("coord_x", double_t, 0, "x", 0,  -2, 2)
gen.add("coord_y", double_t, 0, "y",    0, -2,   2)
gen.add("coord_z", double_t, 0, "z", 0,  -2, 2)
gen.add("is_change", bool_t, 0, "is",    False)

exit(gen.generate(PACKAGE, "mrm_env", "DynReconf"))
